index.html

```diff
@@ -105,10 +110,11 @@ <h2 id="colored">I'm a Software Developer</h2>
  <div id="projects-section">
     <section id="intro">
       <h1>My Recent <br>Works</h1>
+    </section>
       <h2 id="line">
         <img src="images/ind.jpg" alt="_">
       </h2>
-    </section>
+    
 <div id="projects">
```

Comment:
Kindly add some margin to the sides of the projects section, also the cards are not matching the Figma design. 
| Your design  | Figma design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/33268927/173056627-2bb5d8f7-2a33-4e13-b730-29ecb266bfdc.png) | ![image](https://user-images.githubusercontent.com/33268927/173056527-dd63cfc3-366d-4271-ad7b-777e0cddeb33.png) |